### ISSUE TITLE
Authorization header check only in one variant

### ISSUE THREAD

**Issue by team55** [devicehive/esp8266-firmware#42]
модуль httpd.c
В rest запросах заголовок Authorization проверяется только на вариант 'Authorization', тогда как многие клиенты преобразуют его в нижний регистр. 
К примеру так делает react-native при работе через библиотеку okhttp, и на это нельзя повлиять..
Разный регистр заголовков вполне соответствует стандарту
https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2

Логичнее было бы и проверять заголовки без привязки к регистру (либо преобразовать верхний-нижний)

**Comment by team55** [comment]
https://github.com/facebook/react-native/issues/16988 думал проблема в ReactNative

**Comment by Nikolay-Kha** [comment]
Да... по спекам хедеры могут быть в любом регистре. Поправим. Спасибо.

**Comment by Nikolay-Kha** [comment]
Исправлено. Дайте знать если нужна бинарная сборка.
